engine/classes/Elgg/I18n/Translator.php

```diff
@@ -18,6 +18,15 @@ class Translator {
 	private $CONFIG;
 
 	/**
+	 * Regexp for finding keyword arguments in language strings.
+	 * Accepts forms like {label}, { label}, and {label } where label can include
+	 * a-z, A-Z, 0-9, _, -, :, /, and .
```

Comment:
Though thinking about it, we were using it for a very specific system that used namespaced _keywords_. That's probably not a common use case, so this can just be `:alnum:`.